packages/block-library/src/media-text/index.js

```diff
@@ -66,6 +66,9 @@ const blockAttributes = {
 		type: 'boolean',
```

Comment:
It would be nice to add another block fixture in packages/e2e-tests/fixtures/blocks that test a media-text block with a left or right alignment.

Reply:
Recently noticed this comment when scrolling back through the comment history. Added a fixture that tests for the vertical alignment.